new main

BramHouben opened the pull request:
new main

FritsvanLieshout committed:
Added windturbine parks to the map

ramonaerts committed:
changed weather to send the coordinates instead of city

ramonaerts committed:
refactored code to use new API + removed old weather pictures in assets folder

ramonaerts committed:
added minor bug

ramonaerts committed:
implemented weather forecast

Commit:
region page added

ramonaerts committed:
added grade icons to weather forecast

ramonaerts committed:
added wind direction to weather forecast

ramonaerts committed:
merge dev into weatherexpectation

ramonaerts committed:
fixed code smells

Commit:
region now working with paging and has a map

FritsvanLieshout committed:
Added Modal to add scenario's

FritsvanLieshout committed:
Added extra turbine info

Commit:
added count package for idea. and new implementation region details

FritsvanLieshout committed:
Merge pull request #25 from BramHouben/feature/15-weatherexpectation

Feature/15 weatherexpectation

imadfetouh committed:
grid layout fix

imadfetouh committed:
mobile

Commit:
refresh works and region page is now usable

FritsvanLieshout committed:
Fixed charts

FritsvanLieshout committed:
Added some properties to chart

Commit:
name changes and english dutch text

BramHouben committed:
Merge branch 'dev' into regionfilter

Commit:
fixes

Commit:
Merge branch 'regionfilter' of https://github.com/BramHouben/energygrid-east-webapp into regionfilter

FritsvanLieshout committed:
Fixen get form data

Commit:
api changes

Robert8034 committed:
Merge pull request #27 from BramHouben/regionfilter

Regionfilter

Robert8034 committed:
Merge branch 'dev' into grid-fix

Robert8034 committed:
Merge pull request #26 from BramHouben/grid-fix

grid layout fix

FritsvanLieshout committed:
Fixed formdata

FritsvanLieshout committed:
Finished charts

FritsvanLieshout committed:
Merge branch 'dev' into feature/44_wind-turbines

FritsvanLieshout committed:
Last changes

snyk-bot committed:
fix: upgrade multiple dependencies with Snyk

Snyk has created this PR to upgrade:
  - react from 17.0.1 to 17.0.2.
    See this package in npm: https://www.npmjs.com/package/react
  - react-dom from 17.0.1 to 17.0.2.
    See this package in npm: https://www.npmjs.com/package/react-dom

See this project in Snyk:
https://app.snyk.io/org/bramhouben/project/14fc269d-9360-4616-ab9c-749abd5bead2?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade react-i18next from 11.8.10 to 11.8.11

Snyk has created this PR to upgrade react-i18next from 11.8.10 to 11.8.11.

See this package in npm:
https://www.npmjs.com/package/react-i18next

See this project in Snyk:
https://app.snyk.io/org/bramhouben/project/14fc269d-9360-4616-ab9c-749abd5bead2?utm_source=github&utm_medium=upgrade-pr

FritsvanLieshout committed:
FIxes

FritsvanLieshout committed:
Fixed some issues from Sonar

Robert8034 committed:
Merge pull request #30 from BramHouben/feature/44_wind-turbines

Feature/44 wind turbines

BramHouben committed:
Merge branch 'dev' into snyk-upgrade-85ad40a644b102832d04e25705121708

BramHouben committed:
Merge pull request #29 from BramHouben/snyk-upgrade-85ad40a644b102832d04e25705121708

[Snyk] Upgrade react-i18next from 11.8.10 to 11.8.11

BramHouben committed:
Merge branch 'dev' into snyk-upgrade-12d593eb3de457aa84c52f5cc180c642

Robert8034 committed:
Merge pull request #28 from BramHouben/snyk-upgrade-12d593eb3de457aa84c52f5cc180c642

[Snyk] Upgrade: react, react-dom

Robert8034 commented on src/components/shared/regionfilter/housetable/index.jsx:
```diff
@@ -0,0 +1,155 @@
+    this.fetchData(this.state.page);
+  }
+  render() {
+    const { t } = this.props;
```
```suggestion
```
Volgens sonarcloud wordt deze var nooit gebruikt, klopt dat?

Robert8034 replied:
@BramHouben Dit is jouw commit, kan je het uitleggen?

BramHouben replied:
Ja ik kreeg die translation niet gefixt in deze pagina

snyk-bot committed:
fix: upgrade react-tooltip from 4.2.15 to 4.2.17

Snyk has created this PR to upgrade react-tooltip from 4.2.15 to 4.2.17.

See this package in npm:
https://www.npmjs.com/package/react-tooltip

See this project in Snyk:
https://app.snyk.io/org/bramhouben/project/14fc269d-9360-4616-ab9c-749abd5bead2?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade @testing-library/jest-dom from 5.11.9 to 5.11.10

Snyk has created this PR to upgrade @testing-library/jest-dom from 5.11.9 to 5.11.10.

See this package in npm:
https://www.npmjs.com/package/@testing-library/jest-dom

See this project in Snyk:
https://app.snyk.io/org/bramhouben/project/14fc269d-9360-4616-ab9c-749abd5bead2?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade react-map-gl from 6.1.10 to 6.1.11

Snyk has created this PR to upgrade react-map-gl from 6.1.10 to 6.1.11.

See this package in npm:
https://www.npmjs.com/package/react-map-gl

See this project in Snyk:
https://app.snyk.io/org/bramhouben/project/14fc269d-9360-4616-ab9c-749abd5bead2?utm_source=github&utm_medium=upgrade-pr

BramHouben committed:
Merge branch 'dev' into snyk-upgrade-262db8f504f055864557b3d1a45dc7fe

BramHouben committed:
Merge branch 'dev' into snyk-upgrade-7a3ec04d6184d7b12760503f34dee557

BramHouben committed:
Merge branch 'dev' into snyk-upgrade-8992f112097cc243e212122cc535acd6

snyk-bot committed:
fix: upgrade @testing-library/react from 11.2.5 to 11.2.6

Snyk has created this PR to upgrade @testing-library/react from 11.2.5 to 11.2.6.

See this package in npm:
https://www.npmjs.com/package/@testing-library/react

See this project in Snyk:
https://app.snyk.io/org/bramhouben/project/14fc269d-9360-4616-ab9c-749abd5bead2?utm_source=github&utm_medium=upgrade-pr

BramHouben committed:
Merge branch 'dev' into snyk-upgrade-a06064f421f4c1d4ca57b3c5fd1793be

FritsvanLieshout committed:
Merge pull request #33 from BramHouben/snyk-upgrade-262db8f504f055864557b3d1a45dc7fe

[Snyk] Upgrade react-tooltip from 4.2.15 to 4.2.17

BramHouben committed:
Merge branch 'dev' into snyk-upgrade-8992f112097cc243e212122cc535acd6

BramHouben committed:
Merge branch 'dev' into snyk-upgrade-7a3ec04d6184d7b12760503f34dee557

BramHouben committed:
Merge branch 'dev' into snyk-upgrade-a06064f421f4c1d4ca57b3c5fd1793be

Robert8034 committed:
Merge pull request #37 from BramHouben/snyk-upgrade-a06064f421f4c1d4ca57b3c5fd1793be

[Snyk] Upgrade @testing-library/react from 11.2.5 to 11.2.6

Robert8034 committed:
Merge branch 'dev' into snyk-upgrade-8992f112097cc243e212122cc535acd6

BramHouben committed:
Merge branch 'dev' into snyk-upgrade-7a3ec04d6184d7b12760503f34dee557

BramHouben committed:
Merge pull request #36 from BramHouben/snyk-upgrade-8992f112097cc243e212122cc535acd6

[Snyk] Upgrade react-map-gl from 6.1.10 to 6.1.11

BramHouben committed:
Merge branch 'dev' into snyk-upgrade-7a3ec04d6184d7b12760503f34dee557

Robert8034 committed:
Merge pull request #34 from BramHouben/snyk-upgrade-7a3ec04d6184d7b12760503f34dee557

[Snyk] Upgrade @testing-library/jest-dom from 5.11.9 to 5.11.10

Commit:
translation and rework login and register

Commit:
Merge branch 'dev' of https://github.com/BramHouben/energygrid-east-webapp into dev
